server_action_navigate/models/ir_actions_server.py

```diff
@@ -9,7 +9,9 @@
 class IrActionsServer(models.Model):
     _inherit = "ir.actions.server"
 
-    state = fields.Selection(selection_add=[("navigate", "Navigate")])
+    state = fields.Selection(
+        selection_add=[("navigate", "Navigate")], ondelete={"navigate": "set default"}
```

Comment:
I never saw ondelete on Selection fields. What is the interest, changing value when the module is uninstalled ?

Reply:
> 

Here I have added a new selection value to an existing field. So if the module gets uninstall, then it sets the default value, so that for the records that don't have the new value, get the default value set instead of the blank value.

Reply:
but it doesn't make sense. to have a "navigate" ir.actions.server with a state != "navigate". 
If we uninstall the module, a better solution is to delete the ``ir.actions.server`` row where ``state == 'navigate'``.
That can be done in an uninstall_hook.

don't you think ?

Reply:
I have updated the change you suggested to remove the rows with uninstall hook in the last commit, but odoo is refusing the module without the ondelete param, you can see the error log here [https://postimg.cc/svcHyDNs] while installing the module. So I haven't removed the ondelete parameter as of now.

Reply:
@legalsylvain what do you think regarding this update?

Reply:
LGTM, Thanks !